server/etcdmain/config_test.go

```diff
@@ -395,6 +397,55 @@ func TestFlagsPresentInHelp(t *testing.T) {
 	})
 }
 
+func TestParseFeatureGateFlags(t *testing.T) {
```

Comment:
Should we also update `tests/e2e/etcd_config_test.go` to add a feature gate related case? Might be nice to verify something like feature gate in config file.

Reply:
That is a good suggestion. But right now, the features are not actually used, so the e2e would have no visible effect. In the followup PR that uses `StopGRPCServiceOnDefrag` we will have a e2e test added. Is that ok?

Reply:
No problem, just want to make sure e2e suite will also have coverage to exercise feature flag related paths in future.  I'm happy to defer it to next planned pr.